python/meep.i

```diff
@@ -418,11 +418,14 @@ meep::volume_list *make_volume_list(const meep::volume &v, int c,
 template<typename dft_type>
 PyObject *_get_dft_array(meep::fields *f, dft_type dft, meep::component c, int num_freq) {
     int rank;
-    size_t dims[3];
+    size_t dims[3] = {1,1,1};
```

Comment:
This is fine, but the old code seemed safe — `get_dft_array` initializes only `rank` elements of `dims`, but those are all of the elements that we actually use.